src/Settings/settings.json

```
@@ -1176,5 +1176,11 @@
     "type": "bool",
     "default": "false",
     "tip": "Automatically uncheck test cases when they get accepted."
+  },
+  {
+    "name": "On Full Screen Dialog Shown",
```

Any issues with this name?
```suggestion
    "name": "Full Screen Dialog Shown",
```

---

Changed